### ISSUE TITLE
[mdoc][bug] Two assemblies with the same namespace, class and member, but different signatures skip output in ECMAXML

### ISSUE THREAD

**Issue by dend** [mono/api-doc-tools#15]
The scenario is that you have two frameworks: `DenDeliFramework` and `DenDeliBestFramework` - both have their respective assemblies that can have different names, but within them have exactly the same namespace > type > member mapping.

So, say in `DenDeliFramework` I have (represented by `dendeliframework.dll`):

```
namespace DenDeliFramework
{
    public class President
    {
        public string GetInformation(string requestor)
        {
            string result = $"Requestor was {requestor}";
            return result;
        }
    }
}
```

In `DenDeliBestFramework` I have (represented by `dendelibestframework.dll`):

```
namespace DenDeliFramework
{
    public class President
    {
        public string GetInformation(string requestor, string meta)
        {
            string result = $"Requestor was {requestor} and {meta}";
            return result;
        }
    }
}
```

Notice that in the second implementation, `GetInformation` has a different signature. So once I run `mdoc`, here is the output:

```
<Type Name="President" FullName="DenDeliFramework.President">
  <TypeSignature Language="C#" Value="public class President" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit President extends System.Object" />
  <AssemblyInfo>
    <AssemblyName>DenDeliBestFramework</AssemblyName>
    <AssemblyVersion>1.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>DenDeliFramework</AssemblyName>
    <AssemblyVersion>1.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Object</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>To be added.</summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public President ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyVersion>1.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>To be added.</summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName="GetInformation">
      <MemberSignature Language="C#" Value="public string GetInformation (string requestor);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig instance string GetInformation(string requestor) cil managed" />
      <MemberType>Method</MemberType>
      <AssemblyInfo>
        <AssemblyVersion>1.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <ReturnValue>
        <ReturnType>System.String</ReturnType>
      </ReturnValue>
      <Parameters>
        <Parameter Name="requestor" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="requestor">To be added.</param>
        <summary>To be added.</summary>
        <returns>To be added.</returns>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
  </Members>
</Type>
```

Notice two things:

1. The `AssemblyName` is dropped from the `GetInformation` member as well as the constructor.
2. The `GetInformation` reflection in XML shows the implementation of the signature from `DenDeliFramework` but completely skipped over `DenDeliBestFramework`.

This scenario will be especially critical as we try to scale reflection across .NET Framework, .NET Core, .NET Standard and Xamarin BCL since there is a chance some implementations might differ.

**Comment by joelmartinez** [comment]
Yes ... so this is closely tied to #13. Technically, the functionality to handle this precise scenario is in the mdoc codebase, however it's not exposed for regular `mdoc update` use. Once #13 is implemented, it will essentially resolve this issue :)

**Comment by dend** [comment]
@joelmartinez awesome! Just want to make sure that we track all issues 👍

**Comment by joelmartinez** [comment]
I'm going to close this item, as it is resolved by using the `-frameworks` functionality, implemented in #13, which is available in the latest [releases](https://github.com/mono/api-doc-tools/releases)
